config_generation/db_to_xml.py

```diff
@@ -148,35 +148,265 @@ def convert_template_to_scraper(self, collection) -> None:
         scraper_config = self.update_config_xml()
         return scraper_config
 
-    def convert_template_to_plugin_indexer(self, scraper_editor) -> None:
+    def convert_template_to_job(self, collection, job_type) -> None:
         """
-        assuming this class has been instantiated with the scraper_template.xml
+        assuming this class has been instantiated with the job_template.xml
+        """
+        if job_type == "scrapers":
+            self.update_or_add_element_value("Collection", f"/scrapers/{collection.config_folder}/")
+        elif job_type == "indexer":
+            self.update_or_add_element_value("Collection", f"/SDE/{collection.config_folder}/")
+        job_config = self.update_config_xml()
+        return job_config
+
+    def convert_template_to_indexer(self, scraper_editor) -> None:
+        """
+        assuming this class has been instantiated with the final_config_template.xml
         """
 
         transfer_fields = [
-            "KeepHashFragmentInUrl",
+            "connector",
+            "description",
```

Comment:
I see several of these that should not be in the list, such as workercount, index, indexers, identity, description, treeroot, visibility, maxtocrawl, maxlevel, etc. Can we take a more in depth look at this list and only include the ones that need to be transferred. 

Keep in mind that many fields should be defined in the template and should never change on an indexer by indexer basis, such as ExactDeduplication and NearDeduplication. Other fields are always blank, or are server specific, such as identity. These fields should not be in this list.

You can double check me, but I think all that should go here are things that potentially affect the previews or the ability to index difficult pages.